packages/edit-site/src/store/selectors.js

```diff
@@ -80,6 +97,7 @@ export const getSettings = createSelector(
 			focusMode: isFeatureActive( state, 'focusMode' ),
 			hasFixedToolbar: isFeatureActive( state, 'fixedToolbar' ),
 			__experimentalSetIsInserterOpened: setIsInserterOpen,
+			__experimentalReusableBlocks: getReusableBlocks( state ),
```

Comment:
For me the whole getSettings selector is weird, it should only return `state.settings` (edit-site settings), the rest seems more some component related logic to me (useMemo) but I guess it was like that before this PR anyway

Reply:
For reference here are [editor settings for the `@wordpress/editor` package](https://github.com/WordPress/gutenberg/blob/d1a34ef0a4c6b2530e2cd0c91da2d45228a7a17d/packages/editor/src/components/provider/use-block-editor-settings.js#L90).